references/cli/system.go

```diff
@@ -0,0 +1,175 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/oam-dev/cluster-gateway/pkg/generated/clientset/versioned"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/multicluster"
+	"github.com/oam-dev/kubevela/pkg/utils/common"
+)
+
+const (
+	// FlagDeploymentName specifies the deployment name
+	FlagDeploymentName = "name"
+)
+
+// NewSystemCommand print system detail info
+func NewSystemCommand(c common.Args) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "system",
+		Short: "Manage system.",
+		Long:  "Manage system, incluing printing the system deployment information in vela-system namespace and diagnosing the system's health.",
+		Example: "# Check all deployments information:\n" +
+			"> vela system info\n" +
+			"# Specify a deployment name to check detail information:\n" +
+			"> vela system info --name kubevela-vela-core\n" +
+			"# Diagnose the system's health:\n" +
+			"> vela system diagnose\n",
+		Annotations: map[string]string{
+			types.TagCommandType: types.TypeSystem,
+		},
+	}
+	cmd.AddCommand(
+		NewSystemInfoCommand(c),
+		NewSystemDiagnoseCommand(c))
+	return cmd
+}
+
+// NewSystemInfoCommand prints system detail info
+func NewSystemInfoCommand(c common.Args) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "info",
+		Short: "Print the system deployment detail information in vela-system namespace.",
+		Long:  "Print the system deployment detail information in vela-system namespace.",
+		Args:  cobra.ExactValidArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Get deploymentName from flag
+			deployName, err := cmd.Flags().GetString(FlagDeploymentName)
+			if err != nil {
+				return errors.Wrapf(err, "failed to get deployment name flag")
+			}
+			table := newUITable().AddRow("NAME", "NAMESPACE", "IMAGE", "ARGS")
+			table.MaxColWidth = 120
+			config, err := c.GetConfig()
+			if err != nil {
+				return err
+			}
+			// Get clientset
+			clientset, err := kubernetes.NewForConfig(config)
+			if err != nil {
+				panic(err)
+			}
+			// Get deploymentsClient
+			deploymentsClient := clientset.AppsV1().Deployments(types.DefaultKubeVelaNS)
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			if deployName != "" {
+				// DeployName is not empty, print the specified deployment's all args
+				deployment, err := deploymentsClient.Get(ctx, deployName, metav1.GetOptions{})
+				if err != nil {
+					panic(err)
+				}
+				table.AddRow(deployment.Name, deployment.Namespace, deployment.Spec.Template.Spec.Containers[0].Image, strings.Join(deployment.Spec.Template.Spec.Containers[0].Args, " "))
```

Comment:
If we are getting a single deployment, which means the user want some detailed info, is fitting that deployment in a single row a good choice?